test/extended/networking/commatrix.go

```diff
@@ -0,0 +1,185 @@
+package networking
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	"github.com/sirupsen/logrus"
+
+	"github.com/openshift-kni/commatrix/pkg/client"
+	"github.com/openshift-kni/commatrix/pkg/types"
+
+	configv1 "github.com/openshift/api/config/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/openshift-kni/commatrix/pkg/utils"
+	exutil "github.com/openshift/origin/test/extended/util"
+
+	commatrixcreator "github.com/openshift-kni/commatrix/pkg/commatrix-creator"
+	"github.com/openshift-kni/commatrix/pkg/endpointslices"
+	listeningsockets "github.com/openshift-kni/commatrix/pkg/listening-sockets"
+	matrixdiff "github.com/openshift-kni/commatrix/pkg/matrix-diff"
+	configv1client "github.com/openshift/client-go/config/clientset/versioned/typed/config/v1"
+)
+
+const (
+	serviceNodePortMin = 30000
+	serviceNodePortMax = 32767
+)
+
+var _ = g.Describe("[sig-network][Feature:commatrix][Serial]", func() {
+	g.It("should validate the communication matrix ports match the node's listening ports", func() {
```

Comment:
Is it possible to add `[apigroup:config.openshift.io]` to the name so that it is automatically skipped for MicroShift?